public/app/project-setup/theme/mixins/_section.scss

```diff
@@ -2,9 +2,9 @@
 // $type - The type of the section: 'dialog' for dialogs. Default: false.
 @mixin section($type: false) {
   @if $type == dialog {
-    padding: $column-width 0 $column-width 0;
+    padding: $global-padding 0 $global-padding 0;
   }
   @else {
-    padding: $column-width;
+    padding: $global-padding;
```

Comment:
Why don't you use `column-width` anymore? It's actually a design requirement to have the padding be one column width. I find it more expressive to say `column-width` here. What does `global-padding` stand for? What's its semantics?